qutip/core/cy/qobjevo.pyx

```diff
@@ -1,5 +1,6 @@
-#cython: language_level=3
-#cython: boundscheck=False, wraparound=False, initializedcheck=False, cdvision=True
+# cython: language_level=3
+# cython: boundscheck=False, wraparound=False,
+#         initializedcheck=False, cdvision=True
```

Comment:
These are cython options, not comments.
Splitting it like this change the behavior!